iphone/TitaniumKit/TitaniumKit/Sources/API/KrollBridge.m

```diff
@@ -692,8 +691,7 @@ - (KrollWrapper *)loadCommonJSModule:(NSString *)code withSourceURL:(NSURL *)sou
   [eval release];
 
   if (exception != NULL) {
-    id excm = [KrollObject toID:context value:exception];
-    [[TiExceptionHandler defaultExceptionHandler] reportScriptError:[TiUtils scriptErrorValue:excm]];
+    [TiExceptionHandler.defaultExceptionHandler reportScriptError:exception inKrollContext:context];
```

Comment:
@janvennemann I think two methods you introduced in TiExceptionHandler are not required. You can get TiScriptError by using the new method in TiUtils. Something like below -

[TiExceptionHandler.defaultExceptionHandler reportScriptError:[TiUtils scriptErrorFromValueRef: exception inContext: context.context]];

Reply:
Sure, they are not required. However, i added them purely for convenience reasons since we have different source types that we need to create `TiScriptError` from. I didn't like repeatedly having to use `[TiUtils scriptErrorFromValueRef: exception inContext: context.context]]` all over the place so i moved that into the two new methods.

IMHO `[TiExceptionHandler.defaultExceptionHandler reportScriptError:exception inKrollContext:context];` looks nicer and is easier to read than having a nested method. Another benefit is that we only need to adjust the `TiUtils` usage in one place should we need to change this again.